src/utils/savedFilters.ts

```diff
@@ -67,3 +72,36 @@ export const getFilterIDs = (json) => {
     traverse(json);
     return result;
 };
+
+export const errorHandler = (e, res) => {
```

Comment:
Since the error process a particular error I would name the func accordingly:

notUniqueErrorHandler or something similar.

Reply:
I named it generic in case we want to add different errors handling in here